### ISSUE TITLE
Aria-expanded should be set to "false" when the dropdown is not open

### ISSUE THREAD

**Issue by adelriosantiago** [sagalbot/vue-select#1027]
**Describe the bug**
According to the accesibility standards. The "aria-expanded" should be set to "false" when the dropdown is not open.

**To Reproduce**
1. Go to https://vue-select.org/ and see the selector's HTML in the console log. For your convenience here it is:
![image](https://user-images.githubusercontent.com/5462382/70746018-084b3a80-1cf3-11ea-9126-5061ae1b1358.png)

**Expected behavior**
The element should contain the attribute `aria-expanded="false"` before it is even opened. (Of course, it should go to "true" when opened and back to "false" when closed again)

**Desktop (please complete the following information):**
 - OS: Any
 - Browser: All

**Comment by sagalbot** [comment]
Thanks for reporting! Fixed in the v3.5.2 release.
